chains/arbitrum-sepolia-testnet.json

```diff
@@ -0,0 +1,30 @@
+{
+  "alias": "arbitrum-sepolia-testnet",
+  "blockTimeMs": 257,
+  "decimals": 18,
+  "explorer": {
+    "api": {
+      "key": {
+        "hardhatEtherscanAlias": "arbitrumSepolia",
+        "required": true
+      },
+      "url": "https://api-sepolia.arbiscan.io/api"
+    },
+    "browserUrl": "https://sepolia.arbiscan.io/"
+  },
+  "id": "421614",
+  "name": "Arbitrum Sepolia testnet",
+  "providers": [
+    {
+      "alias": "default",
+      "rpcUrl": "https://sepolia-rollup.arbitrum.io/rpc"
+    },
+    {
+      "alias": "publicnode",
+      "rpcUrl": "https://arbitrum-sepolia-rpc.publicnode.com"
+    }
+  ],
+  "skipProviderCheck": true,
```

Comment:
Why skip the provider check for all of these? We're skipping it for all goerli related chains, but none otherwise, including e.g. Blast sepolia:
https://github.com/api3dao/chains/blob/da3050a175304fe3cbb9c61771a68f039840dd3e/src/generated/chains.ts#L206-L208